dojo/tools/api_blackduck/api_client.py

```diff
@@ -2,8 +2,8 @@
 
 # Ignore DeprecationWarning from the blackduck library because we cannot do anything about it
 # until they fix it in their library
-warnings.filterwarnings("ignore", category=DeprecationWarning)
-from blackduck import Client  # noqa: E402
+with warnings.catch_warnings(action="ignore", category=DeprecationWarning):
```

Comment:
Can we add here `# TODO` with a link to the GitHub issue where this problem was raised?